app/controllers/boxes_controller.rb

```diff
@@ -146,7 +146,7 @@ def load_box_samples
     samples = if @box.blinded? && !params[:unblind]
       samples.scrambled
     else
-      samples.sort_by{ |sample|  [ sample.batch_number.to_s, sample.concentration , sample.replicate ] }
+      samples.sort_by{ |sample|  [ sample.id ] }
```

Comment:
We don't need to load the samples to sort them anymore (no more entity framework field):

```suggestion
      samples.order(:id)
```